sdk-core/src/main/java/care/data4life/sdk/tags/TagHelper.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2020 D4L data4life gGmbH / All rights reserved.
+ *
+ * D4L owns all legal rights, title and interest in and to the Software Development Kit ("SDK"),
+ * including any intellectual property rights that subsist in the SDK.
+ *
+ * The SDK and its documentation may be accessed and used for viewing/review purposes only.
+ * Any usage of the SDK for other purposes, including usage for the development of
+ * applications/third-party applications shall require the conclusion of a license agreement
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+package care.data4life.sdk.tags
+
+import java.util.*
+
+object TagHelper {
+
+    private const val TAG_DELIMITER = "="
+
+    @JvmStatic
+    fun convertToTagList(tags: HashMap<String, String>): List<String> {
+        val tagList: MutableList<String> = ArrayList()
+        for ((key, value) in tags) {
+            tagList.add(key + TAG_DELIMITER + value)
+        }
+        return tagList
+    }
```

Comment:
Just as a matter of taste:
``` kotlin
fun convertToTagList(
    tags: HashMap<String, String>
): List<String> = arrayListOf().also {
        for ((key, value) in tags) {
            it.add(key + TAG_DELIMITER + value)
        }
}
```

Reply:
I move the return

```
fun convertToTagList(tags: HashMap<String, String>): List<String> {
        return mutableListOf<String>().also {
            for ((key, value) in tags) {
                it.add(key + TAG_DELIMITER + value)
            }
        }
}
```

Reply:
Did you changed it? I looks like before? However, I am also fine with the version no-also version, but I am not sure, which you intended to use actually.